fitbenchmarking/resproc/visual_pages.py

```diff
@@ -67,100 +65,33 @@ def create(prob_results, group_name, results_dir, count):
                       key=lambda result: result.chi_sq
                       if result.chi_sq is not np.nan else np.inf)
 
-    problem_name = process_problem_name(best_result.problem.name)
-    support_pages_dir, file_path, see_also_link = setup_page_misc(
-        group_name, problem_name, best_result, results_dir, count)
-    rst_link = generate_rst_link(file_path)
-
-    ini_details_tbl, fin_details_tbl = \
-        setup_detail_page_tbls(best_result.ini_function_def,
-                               best_result.fin_function_def)
+    prob_name = best_result.problem.name
+    prob_name = prob_name.replace(',', '')
+    prob_name = prob_name.replace(' ', '_')
 
+    support_pages_dir, file_path, see_also_link = \
```

Comment:
If we're not using the `see_also_link` this could be removed in the function, and the function could be given a more descriptive name ("get_filename_and_path" or something rather than "misc")

Reply:
done